### PR TITLE
added md5exception

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.jalasoft.jfc.model.exception;
+
+/**
+ * Class of manage exception of Md5.
+ *
+ * @version 0.1 03 Jan 2020.
+ *
+ * @author Alan Escalera.
+ */
+
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java]: delete this line of code please

Comment (Amet015) [reply src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java]: ok

File: src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Class of manage exception of Md5.
+ *
+ * @version 0.1 03 Jan 2020.
+ *
+ * @author Alan Escalera.
+ */
+public class Md5Exception extends Exception {
+
+        // Returns a message error.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java]: improve this comment to "Returns a error message"

**Review by JuanCarlosMartinez4 — APPROVED** [review]

File: src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java (hunk 2 above)
Comment (Amet015) [reply src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java]: ok, it is already fixed.

**Pull request merged**